doc/nrf/device_guides/working_with_nrf/nrf70/developing/constrained.rst

```diff
@@ -291,6 +311,7 @@ The nRF70 Series driver can be used in the following profiles (not an exhaustive
        ``UDP-RX: 12.7 Mbps``
 
 .. note::
-   The measured throughputs, as shown in the table above, are based on tests conducted using the nRF7002 DK. The results represent the best throughput, averaged over three iterations, and were obtained with a good RSSI signal in a clean environment.
+   The measured throughputs, as shown in the table above, are based on tests conducted using the nRF7002DK.
```

Comment:
We write DK with a space.
```suggestion
   The measured throughputs, as shown in the table above, are based on tests conducted using the nRF7002 DK.
```

Reply:
Well, this was based on a previous comment of yours :).

Reply:
Sorry, no, you suggested space earlier too.